src/App.js

```diff
@@ -9,7 +10,7 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article props={article} />
```

Comment:
Don't use `props` as prop name, it's very confusing.
Also, it`s better to pass separate params rather than whole `article` object.

Reply:
Done!